### PR TITLE
Material UI addition

### PR DESCRIPTION
**Pull request by imyint**

### CODE REVIEW

File: client/src/components/shared.tsx [hunk 1]
```diff
@@ -166,6 +167,24 @@ export const CenteredButton = styled.button`
   }
 `;
 
+export const CenteredButton = styled.button`
```
Comment (timothycobrien) [inline client/src/components/shared.tsx:170]: Why use this over material Button?

File: client/src/components/leadership.tsx [hunk 2]
```diff
@@ -1,7 +1,127 @@
...
-    return(<h1 style={{"gridArea": "main", "padding": "10px 50px", "color": "white"}}>Leadership Board for {currentUser}</h1>);
+    const classes = useStyles();
+    return(<LeaderBoardBlockBase className = {classes.backDrop}>
+        <Header> LEADERSHIP BOARD </Header>
+        <TableContainer className={classes.table}>
+            <Table className={classes.table} aria-label="customized table">
+                <TableHead>
+                    <TableRow className={classes.tableHeader}>
+                        <StyledTableCell align="left">Rank</StyledTableCell>
+                        <StyledTableCell align="right"></StyledTableCell>
```
Comment (timothycobrien) [inline client/src/components/leadership.tsx:104]: Is this to linebreak?

**Review by timothycobrien — CHANGES_REQUESTED** [review]

File: client/src/components/shared.tsx (hunk 1 above)
Comment (imyint) [reply client/src/components/shared.tsx:170]: idk what this is, I just didnt want to accidentally remove Sam's code

File: client/src/components/leadership.tsx (hunk 2 above)
Comment (imyint) [reply client/src/components/leadership.tsx:104]: To skip over that column

Comment (timothycobrien) [reply client/src/components/leadership.tsx:104]: That's fine

**Review by timothycobrien — APPROVED** [review]

**Pull request merged**